includes/Core/User_Input/Site_Specific_Questions.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\User_Input\Site_Specific_Questions
+ *
+ * @package   Google\Site_Kit\Core\User_Input
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\User_Input;
+
+use Google\Site_Kit\Core\Storage\Setting;
+
+/**
+ * Class for handling the site specific settings in User Input.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Site_Specific_Questions extends Setting {
+
+	/**
+	 * The option_name for this setting.
+	 */
+	const OPTION = 'googlesitekit_user_input_settings';
+
+	/**
+	 * Gets the expected value type.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string The type name.
+	 */
+	protected function get_type() {
+		return 'array';
+	}
+
+	/**
+	 * Gets the default value.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array The default value.
+	 */
+	protected function get_default() {
+		return array();
+	}
+
+	/**
+	 * Gets the callback for sanitizing the setting's value before saving.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return callable Callback method that filters or type casts invalid setting values.
+	 */
+	protected function get_sanitize_callback() {
+		return function ( $settings ) {
+			return User_Input::sanitize_settings( $this, $settings, 'site' );
```

Comment:
It made sense to keep the `sanitize_settings` in the `User_Input` class when it was called `User_Input_Settings`. but I think it would be better to keep the `sanitized_settings` to the corresponding `**_Specific_Settings` class.

Reply:
Done, thank you!